test/test_foreach.py

```diff
@@ -342,10 +342,10 @@ def _pointwise_test(
                 expected = ref(ref_inputs, values=values)
                 self.assertEqual(expected, actual)
 
-    @dtypes(*all_types_and_complex_and(torch.half, torch.bfloat16, torch.bool))
+    @dtypes(*all_types_and_complex_and(torch.half, torch.bfloat16))
     def test_add_scalar_with_empty_list_and_empty_tensor(self, device, dtype):
         # TODO: enable empty list case
-        for tensors in [[torch.randn([0])]]:
+        for tensors in [[torch.randn([0], device=device, dtype=dtype)]]:
```

Comment:
@kit1980 do you think we can tune lintrunner to somehow find more cases like that in our testsuite?

Reply:
I think there should be a linter to catch this, it's basically unused function argument.
I'll take a look.

Reply:
ruff's ARG001 catches this.

Reply:
It's very noisy on tests though...